frontend/src/store.js

```diff
@@ -0,0 +1,35 @@
+import { createStore, applyMiddleware, compose } from 'redux';
+import { routerMiddleware } from 'react-router-redux';
+import thunk from 'redux-thunk';
+import createHistory from 'history/createBrowserHistory';
+import rootReducer from './modules';
+
+export const history = createHistory();
+
+const initialState = {};
+const enhancers = [];
+const middleware = [
+  thunk,
+  routerMiddleware(history)
+];
+
+if (process.env.NODE_ENV === 'development') {
```

Comment:
is this something that you set up and define somewhere? I don't have to do anything for this, right?